fawltydeps/packages.py

```diff
@@ -309,6 +311,12 @@ class TemporaryPipInstallResolver(BasePackageResolver):
     packages in this venv. The venv is automatically deleted before as soon as
     the packages have been resolved."""
 
+    def __init__(
+        self,
+        cache: Optional[pytest.Cache] = None,
+    ) -> None:
+        self.cache = cache
+
```

Comment:
Instead of this we can use a `cached_venv` class member that defaults to `None`, and that `test_resolver` sets to the cached venv dir before running the test. See below.